students/eowyn/session02/print_grid.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+
+''' Construct grid that is composed of ndim boxes in 
+    either direction, and each box is composed of ndash 
+    lines both horizontally and vertically '''
+
+### First method uses global vars and functions for lines and verts
+### then a caller function to print it all
+
+# define global varaibles
+plus = "+"
+minus = "-"
+space = " "
+dash = '|'
+
+def make_line(ndim,ndash):
+    line = ''
+    for n in range(ndim):
```

Comment:
A loop is fine, but you could get the same effect with multiplication:

line = ndim * (plus + ndash*minus)